### PR TITLE
Add .asn1 dependencies for files generated from providers/common/der/*.in

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: providers/common/der/build.info [hunk 1]
```diff
@@ -3,11 +3,11 @@ $DER_DIGESTS_H=../include/prov/der_digests.h
 $DER_DIGESTS_GEN=der_digests_gen.c
 
 GENERATE[$DER_DIGESTS_GEN]=der_digests_gen.c.in
-DEPEND[$DER_DIGESTS_GEN]=oids_to_c.pm
+DEPEND[$DER_DIGESTS_GEN]=oids_to_c.pm NIST.asn1 DIGESTS.asn1
 
 DEPEND[${DER_DIGESTS_GEN/.c/.o}]=$DER_DIGESTS_H
```
Comment (slontis) [inline providers/common/der/build.info:8]: The blankline should be one line down.

File: providers/common/der/build.info [hunk 2]
```diff
@@ -3,11 +3,11 @@ $DER_DIGESTS_H=../include/prov/der_digests.h
...
 GENERATE[$DER_DIGESTS_H]=der_digests.h.in
-DEPEND[$DER_DIGESTS_H]=oids_to_c.pm
+DEPEND[$DER_DIGESTS_H]=oids_to_c.pm NIST.asn1 DIGESTS.asn1
```
Comment (slontis) [inline providers/common/der/build.info:10]: To what extent do the includ edependancies matter .. e.g this one includes "internal/der.h"

**Review by slontis — APPROVED** [review]

File: providers/common/der/build.info (hunk 2 above)
Comment (levitte) [reply providers/common/der/build.info:10]: Yes.  That is, however, found by the dependency discovery, so no need to make it explicit here

**Comment by levitte** [comment]
Merged

e378be2a29f8bc5e679e63d5f5e9766d2f4dfc4b Add .asn1 dependencies for files generated from providers/common/der/*.in

**Pull request closed without merging**